tslearn/neighbors/neighbors.py

```diff
@@ -1,16 +1,28 @@
 import numpy
+
+from scipy.spatial.distance import cdist as scipy_cdist
+
 from sklearn import neighbors
 from sklearn.neighbors import (KNeighborsClassifier, NearestNeighbors,
                                KNeighborsRegressor)
 from sklearn.utils.validation import check_is_fitted
-from scipy.spatial.distance import cdist as scipy_cdist
 
-from tslearn.metrics import cdist_dtw, cdist_ctw, cdist_soft_dtw, \
-    cdist_sax, TSLEARN_VALID_METRICS
-from tslearn.piecewise import SymbolicAggregateApproximation
-from tslearn.utils import (to_time_series_dataset, to_sklearn_dataset,
-                           check_array, check_dims)
 from tslearn.bases import BaseModelPackage
+from tslearn.metrics import (
+    cdist_dtw,
+    cdist_ctw,
+    cdist_soft_dtw,
+    cdist_sax,
+    cdist_frechet,
+    TSLEARN_VALID_METRICS
+)
+from tslearn.piecewise import SymbolicAggregateApproximation
+from tslearn.utils import (
+    to_time_series_dataset,
+    to_sklearn_dataset,
+    check_array,
+    check_dims
+)
 
 neighbors.VALID_METRICS['brute'].extend(['dtw', 'softdtw', 'sax', 'ctw'])
```

Comment:
Don't we need to add Fréchet here too? Sorry I don't remember why it was important to populate this `VALID_METRICS['brute']` stuff :/